test/unittest/models/models_golden_test.h

```diff
@@ -31,13 +31,16 @@ class NeuralNetwork;
  *
  */
 typedef enum {
-  COMPARE = 1 << 0,           /**< Set this to compare the numbers */
+  NO_THROW_RUN = 0, /**< no comparison, only validate execution without throw */
+  COMPARE = 1 << 0, /**< Set this to compare the numbers */
   SAVE_AND_LOAD_INI = 1 << 1, /**< Set this to check if saving and constructing
                                  a new model works okay (without weights) */
-  COMPARE_V2 = 1 << 2,        /**< compare with v2 model format */
+  USE_V2 = 1 << 2,            /**< use v2 model format */
```

Comment:
USE_V2 vs COMPARE_V2 -> use_v2 just creates model vs compare will compare the values?

Reply:
mode | USE_V2 will run the test on v2 mode. So `compare_v2 = compare | use_v2`

save_load_ini_v2 will be `save_and_load_ini | use_v2`

Reply:
just add a comment then that USE_V2 should not be used by itself.

Reply:
It is allowed and valid to set only USE_V2.

The expected behavior is setting option to 0 but expect v2 behavior.

Reply:
got it.